support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -334,8 +334,8 @@ function sendEventContributionCartValue(
 	});
 }
 
-function sendEventContributionPaymentMethod(
-	paymentMethod: PaymentMethod | null,
+function sendEventPaymentMethodSelected(
+	paymentMethod: PaymentMethod | 'StripeExpressCheckoutElement' | null,
```

Comment:
This `PaymentMethod` type is starting to feel like a code smell, maybe it will become clearer once we remove the old checkouts

Reply:
Yeah agreed, this was copy & paste of a work around applied on the generic checkout, but certainly we could do with revisiting this when we start removing the old checkouts.